webapp/store/views.py

```diff
@@ -62,29 +62,26 @@ def index():
             query=query.lower(), fields=SEARCH_FIELDS
         ).get("results")
 
-        charms = []
-        total_charms = 0
+        packages = []
+        total_packages = 0
 
         for i, item in enumerate(results):
-            if item["type"] != "charm":
-                continue
+            total_packages += 1
 
-            total_charms += 1
-
-            charm = logic.add_store_front_data(
+            package = logic.add_store_front_data(
                 results[i], results[i]["default-release"]
             )
 
-            charms.append(charm)
+            packages.append(package)
 
-        context["results"] = charms
+        context["results"] = packages
         return render_template("store-search.html", **context)
 
     return render_template("store.html", **context)
 
 
-@store.route("/charms.json")
-def get_charms():
+@store.route("/packages.json")
```

Comment:
Maybe @tbille will have to update some of his scripts

Reply:
I am not consuming those APIs so should be fine :smile: